### PR TITLE
Improve accessibility and some minor css fixes

### PR DESCRIPTION
**Pull request by btzr-io**
## PR Checklist

Please check all that apply to this PR using "x":

- [x] I have checked that this PR is not a duplicate of an existing PR (open, closed or merged)
- [x]  I added a line describing my change to CHANGELOG.md
- [x] I have checked that this PR does not introduce a breaking change
- [ ] This PR introduces breaking changes and I have provided a detailed explanation below

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [x] Feature
- [x] Code style update (formatting)
- [x] Refactoring (no functional changes)
- [ ] Documentation changes
- [ ] Other - Please describe:

## Fixes

Issue Number:  #4046, #5991, #6480, #5145, #6483,  #6582, #6635, #6602, #6503
Changelog update: #6735

## Todo

- [x] Use aria-hidden for all icons by default ?
- [x]  Use aria-hidden for some images / thumbnails ? ( https://web.dev/aria-hidden-focus/ )
- [x] Improve tile cards accessibility.
- [x] Improve header accessibility.  ( Skip navigation link https://www.a11ymatters.com/pattern/skip-link/ )
- [ ]  Better css style for skip navigation link ? 
- [x] Change focus order of skip nav link ( Make it first element )
- [x] Fix repost button accessibility

### CODE REVIEW

**Comment by tzarebczan** [comment]
I haven't filed this one, but I remember someone mention accessibility issues around the playlist feature - somewhere in the pop up menus. Can you take a peek?

**Comment by btzr-io** [comment]
> 
> 
> I haven't filed this one, but I remember someone mention accessibility issues around the playlist feature - somewhere in the pop up menus. Can you take a peek?

What issues ? Need more info

**Comment by btzr-io** [comment]
https://www.accessibility-developer-guide.com/

**Comment by btzr-io** [comment]
Skip navigation feature: Useful for screen reader users ( tested with windows narrator )

SkipNavigationLink

**Comment by btzr-io** [comment]
New aria-label format for claim preview links:
This includes additional information of the claim.

```
%title% by %channelTitle%, %mediaDuration%
```

Screen reader
```
Narrator: "Link Awesome Cats! by Animals, one minute (00:10:00)"
```
### Todo

- [ ] Include more info ? ( release date, file type, price ) ?

**Comment by btzr-io** [comment]
Waiting for review or feedback before any more changes.

File: ui/component/header/view.jsx [hunk 1]
```diff
@@ -64,6 +64,26 @@ type Props = {
   activeChannelStakedLevel: number,
 };
 
+// Allow screen reader users ( or keyboard navigation )
+// to jump to main content
+const SkipNavigationButton = () => {
```
Comment (btzr-io) [inline ui/component/header/view.jsx:69]: Should I move this component to is own file ?

**Review by jessopb — APPROVED** [review]

File: ui/scss/component/_button.scss [hunk 2]
```diff
@@ -236,6 +236,7 @@
 
   &:focus {
     box-shadow: none;
+    background-color: var(--color-button-alt-bg);
```
Comment (btzr-io) [inline ui/scss/component/_button.scss:239]: Fix #6503

**Comment by infinite-persistence** [comment]
@btzr-io , can you make one final rebase to resolve the conflicts? Will get this in soon.

File: ui/util/formatAriaLabel.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    }
+  } else {
+    if (ariaDate) {
+      ariaLabelData += ariaLabelData = __('%title% by %channelTitle% %ariaDate%', { title, channelTitle, ariaDate });
+    } else {
+      ariaLabelData += ariaLabelData = __('%title% by %channelTitle%', { title, channelTitle });
+    }
+  }
+
+  return ariaLabelData;
```
Comment (infinite-persistence) [inline ui/util/formatAriaLabel.js:31]: I think we forgot to remove the `+=`?

**Review by infinite-persistence — CHANGES_REQUESTED** [review]
One more minor fix needed

**Comment by btzr-io** [comment]
Changelog update: #6735

**Pull request merged**